ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -204,6 +204,13 @@ tests:
     - ref: openstack-test-openstack
     workflow: openshift-e2e-openstack-ipi
   timeout: 7h0m0s
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 24h
```

Comment:
That's too frequent and not necessary.
Maybe once every two weeks is enough.
I don't think that ingress performance is going to change that much.